paddle/fluid/operators/activation_op.h

```diff
@@ -584,6 +584,72 @@ struct SinFunctor : public BaseActivationFunctor<T> {
   }
 };
 
+template <typename T>
+struct Sinh {
+  HOSTDEVICE T operator()(const T& val) const { return sinh(val); }
+};
+
+template <>
+struct Sinh<platform::float16> {
+  HOSTDEVICE platform::float16 operator()(const platform::float16& val) const {
+    return platform::float16(sinh(static_cast<float>(val)));
```

Comment:
I suggest using `sinhf` instead of `sinh`, because `sinh` receives a `double` and returns a `double`.

Reply:
befer the code, sin and cos function should also be sinf and cosf, Right?

Reply:
done!

Reply:
OK, but since there have been new PRs merged into the repo, this PR now conflicts with the repo. So please pull from the develop branch and push again, thank you.

Reply:
conflicts solved, done.